### ISSUE TITLE
Client Side Virtual Monitor

### ISSUE THREAD

**Issue by dabgdk** [FreeRDP/FreeRDP#6514]
Would you implement client side virtual monitor support?

Suppose the client machine has a single monitor, as does the server. I think it would be cool if the client could be told to tell the server on initial connection that it has say, two monitors. (From a network / protocol perspective, the server would have no idea that the additional monitor is virtual). Once connected, a global hotkey on the client could switch between monitors.

**Comment by dabgdk** [comment]
Additionally, it would be really cool if this could support arbitrary numbers of real and virtual monitors. For example, a client machine with two hardware monitors able to simulate 6 total monitors, and able to dynamically move the virtual monitors between the physical monitors with hotkey controls.

**Comment by akallabeth** [comment]
@dabgdk yes, that could be done on a per client basis (you need to implement that for X11, wayland, ...)
We´re happy about pull requests here, but will not implement this in the foreseeable future.

**Comment by hardening** [comment]
I'd also be super interested by that, I did a quick look and the basics are quite easy (basically bypass xrender detection), but then fixing all the details need a lot of small changes in many places (like breaking multimonitor == fullscreen) assertion.
